.github/CODEOWNERS

```diff
@@ -1,4 +1,5 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-*                                    @MetaMask/devs
+*  @MetaMask/wallet-api-platform-engineers
+*  @MetaMask/tech-writers
```

Comment:
This is technically "valid", but in effect this sets only the `tech-writers` team as the owners. Later lines overwrite earlier ones. They need to be specified on the same line if you want both to be owners

Reply:
Ah, I was afraid that GitHub would do this, thanks for the clarification.